src/main/java/forceitembattle/util/FinishInventory.java

```diff
@@ -2,151 +2,179 @@
+                    List<ForceItem> items = (targetTeam != null ? targetTeam.getFoundItems() : targetPlayer.foundItems());
                     if (items.isEmpty()) {
-                        setItem(startSlot, new ItemBuilder(Material.BARRIER).setDisplayName("§cNo Items found").getItemStack());
+                        setItem(startSlot, new ItemBuilder(Material.BARRIER).setDisplayName("<red>No Items found").getItemStack());
                         placedItems = -1;
                     } else {
                         ForceItem forceItem = items.get(placedItems);
-                        ItemStack itemStack = new ItemBuilder(forceItem.material()).setDisplayName(WordUtils.capitalize(forceItem.material().name().replace("_", " ").toLowerCase()) + (forceItem.usedSkip() ? " §c§lSKIPPED" : "") + " §8» §6" + forceItem.timeNeeded()).setGlowing(forceItem.usedSkip()).getItemStack();
+                        ItemStack itemStack = new ItemBuilder(forceItem.material()).setDisplayName(WordUtils.capitalize(forceItem.material().name().replace("_", " ").toLowerCase()) + (forceItem.usedSkip() ? " <red><b>SKIPPED</b>" : "") + " <dark_gray>» <gold>" + forceItem.timeNeeded()).setGlowing(forceItem.usedSkip()).getItemStack();
                         setItem(startSlot, itemStack);
                         slots.put(startSlot, itemStack);
-                        pages.put(pagesAmount, slots);
                     }
 
                     Bukkit.getOnlinePlayers().forEach(players -> players.playSound(players.getLocation(), Sound.ENTITY_ITEM_PICKUP, 1, 1));
 
                     if(startSlot == 16 || startSlot == 25 || startSlot == 34 || startSlot == 43) startSlot += 3;
                     else startSlot++;
 
-                    if(placedItems >= targetPlayer.foundItems().size() - 1) {
+                    if(placedItems >= (targetTeam != null ? (targetTeam.getFoundItems().size() - 1) : (targetPlayer.foundItems().size() - 1))) {
 
-                        if(pages.isEmpty()) pages.put(0, slots);
+                        pages.put(pagesAmount, new HashMap<>(slots));
 
                         new BukkitRunnable() {
 
                             @Override
                             public void run() {
 
-                                TextComponent placementText = new TextComponent(place + ". " + targetPlayer.player().getName() + " §8┃ §6" + (placedItems + 1) + " Items found §8» ");
-                                TextComponent textComponent = new TextComponent("§8[§bInventory§8]");
-                                textComponent.setClickEvent(new ClickEvent(ClickEvent.Action.RUN_COMMAND, "/result " + targetPlayer.player().getUniqueId()));
+                                String placeColor = "<white>";
+                                if(place == 3) placeColor = "<red>";
+                                else if(place == 2) placeColor = "<gray>";
+                                else if(place == 1) placeColor = "<gold>";

```

Comment:
maybe create some Function for this somewhere un a Util class. Unless you really don't need to use it somewhere else too. It's easier to update the colors in the future. Also you could use switch-case here.